<?xml version="1.0" encoding="ISO-8859-15"?>

<launch>
  <arg name="wait_down_camera" default="false"/>
  <arg name="max_exp" default="10000"/>
  <arg name="min_exp" default="75"/>
  
  <node name="auto_exposure_control" pkg="auto_exposure_control" type="auto_exposure_control.py" respawn="false" output="screen">
    <param name="wait_down_camera" type="bool" value="$(arg wait_down_camera)"/>
    <param name="max_exp" type="int" value="$(arg max_exp)"/>
    <param name="min_exp" type="int" value="$(arg min_exp)"/>
  </node>
  
</launch>
